docs/openapi-docs/src/test/scala/sttp/tapir/docs/openapi/VerifyYamlTest.scala

```diff
@@ -908,6 +908,70 @@ class VerifyYamlTest extends AnyFunSuite with Matchers {
     actualYamlNoIndent shouldBe expectedYaml
   }
 
+  test("should match the expected yaml for circe json value") {
```

Comment:
these are quite heavy tests for a small detail. Maybe we could simply test how schemas look like in the json integrations?

Reply:
ah, yes, that's good idea

Reply:
corrected